ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -100,18 +105,20 @@ def handle(self, *args, **options):
             while True:
                 data = self.from_ejabberd()
                 self.logger.debug("Command is %s" % data[0])
-                if not isinstance(data[1], (int, long)):
-                    success = False
 
-                if data[0] == "auth":
-                    success = self.auth(data[1], data[2], data[3])
-                elif data[0] == "isuser":
-                    success = self.isuser(data[1], data[2])
-                elif data[0] == "setpass":
-                    success = False
+                if self.validate_data_from_ejabberd(data):
```

Comment:
@fichie23 this assume all command require the second member of data is always int, which is possibly not true. I suggest to move this to each of the command, that we know for sure that the second member is required to be int.

Reply:
I think it is @orhan89 , the format always be like this : `command:user_id:server_name:token` , why do you assumed it is not ?

Reply:
i don't assume it's not. i assume it might be change with new command later.